docs/products/opensearch/howto/migrate-ism-policies.md

```diff
@@ -67,6 +76,11 @@ To reapply ISM policies to indices in Aiven for OpenSearch:
    python avn-re-apply-ism-policies.py --config path-to-config-file
    ```
 
+## Re-running the ISM script
+
+You can rerun the ISM script if needed. Use the --force option to bypass the check
+that prevents it from running more than once.
```

Comment:
We should advise to only run this one time after you have finished all the data migration

Reply:
added a note to this section.